spec/lib/debts/letter_downloader_spec.rb

```diff
@@ -8,69 +8,62 @@
 
   let(:file_number) { '796330625' }
   let(:vbms_client) { FakeVbms.new }
-  let(:request_double) do
+
+  def stub_request(request_name, args, return_val)
```

Comment:
I like this method.  Small suggestion, name it something like `stub_vbms_client_request`